webclient/js/elems/raflow/rentables.js

```diff
@@ -742,7 +754,15 @@ window.loadRARentablesGrid = function () {
                                 var BID = getCurrentBID();
                                 app.raflow.arList[BID].forEach(function(item) {
                                     if (event.value_new.id == item.ARID) {
+                                        console.log(item);
```

Comment:
remove/comment console.log if it's no longer used

Reply:
console.log will be removed in final merge in each pull request(It same for future PR).

Reply:
all right